app/views/external_pledges/create.js.erb

```diff
@@ -1 +1 @@
-// rerender #existing-external-pledges
\ No newline at end of file
+$('#external_pledge_amount').val('');
```

Comment:
Also, maybe I should just rerender the partial sorta like how we do with the existing pledges? Figure that might actually be better since the dropdown list would need to lose a value?